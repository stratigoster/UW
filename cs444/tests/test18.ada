PACKAGE foo IS
  TYPE Color IS (Red,Orange,Green,Blue,Indigo,Violet,White,Black);
  TYPE Light IS (Black,Neon,White,Red);
  color : Light := Black;
--^^^^^
-- color conflicts with definition of color at line 3

END foo;
