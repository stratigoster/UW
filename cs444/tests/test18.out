***ERROR***:4:26:	Declaration `color:light' conflicts with `color:enumeration of (red orange green blue indigo violet white black)'

------
Compilation error
------
